templates/etc/apache2/sites-available/apache__tpl_macros.j2

```diff
@@ -65,20 +176,7 @@ DocumentRoot {{ item.root|d(item.document_root) | quote }}
 {{ get_alias(item.alias, item.root|d(item.document_root)) }}
 {%- endif %}
 <Directory {{ item.root|d(item.document_root) | quote }}>
-    Options {{ debops__tpl_macros.get_yaml_list_for_elem(item.options|d(apache__vhost_options))|from_yaml | join(" ") }}
-    AllowOverride {{ debops__tpl_macros.get_yaml_list_for_elem(item.allow_override|d(apache__vhost_allow_override))|from_yaml | join(" ") }}
-
-{% if debops__tpl_macros.get_apache_version() | version_compare("2.4", ">=") %}
-    Require all granted
-{% else %}
-    Order allow,deny
-    Allow from all
-{% endif %}
-{% if item.root_directives|d() %}
-
-    {{ item.root_directives | indent(4) }}
-{% endif %}
-
+    {{ debops__tpl_macros.indent(get_document_root_directives(item), 4) -}}
 </Directory>
```

Comment:
please double check that the closing directive is on a new line.
I've checked out 5e76849c58b0c2439c7d3d0cf9a912cfec68ad14 and it produces the closing Directory on the same line, i.e. `        Require all granted</Directory>`. That makes apache unhappy: `Expected </Directory> but saw </VirtualHost>\n"}`.  That reminds me of https://github.com/debops/ansible-postfix/issues/80